src/main/java/com/projetoGerenciamentoCurso/gerenciamentoCurso/service/AlunoService.java

```diff
@@ -1,36 +1,83 @@
 package com.projetoGerenciamentoCurso.gerenciamentoCurso.service;
 
+import javax.transaction.Transactional;
+
 import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
-import com.projetoGerenciamentoCurso.gerenciamentoCurso.*;
+import com.projetoGerenciamentoCurso.gerenciamentoCurso.GerenciamentoCursoApplication;
 import com.projetoGerenciamentoCurso.gerenciamentoCurso.dto.AlunoDto;
+import com.projetoGerenciamentoCurso.gerenciamentoCurso.models.Aluno;
+import com.projetoGerenciamentoCurso.gerenciamentoCurso.repository.AlunoRepository;
 
 import lombok.AllArgsConstructor;
 
 @Service
 @AllArgsConstructor
+@Transactional
 public class AlunoService {
 
 	private final RabbitTemplate rabbitTemplate;

```

Comment:
Substituir o atributo para uma nova classe chamada "Publisher", que terá responsabilidade exclusiva de enviar eventos
```suggestion
	private final Publisher publisher;
```

Reply:
feito